components/support/migration/src/main/java/mozilla/components/support/migration/Exception.kt

```diff
@@ -10,5 +10,5 @@ package mozilla.components.support.migration
 fun Exception.uniqueId(): String {
     // In extreme cases, VM is allowed to return an empty stacktrace.
     val topStackFrameString = this.stackTrace.getOrNull(0)?.toString() ?: ""
-    return "${this::class.java.canonicalName} $topStackFrameString"
+    return "${this.javaClass.canonicalName} $topStackFrameString"
```

Comment:
i guess this continues to work as designed, right?

Reply:
@grigoryk yeah, it does, but there's also no need to change that here as the original works fine. I will remove.

Reply:
Does the original not rely on the reflection lib? it _looks_ like it does

Reply:
Yeah, it's deceiving but the original actually works, I think `this::class.java` is equivalent to `this.javaClass`.